test/features/visitor/visiting_the_home_page_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+feature "Visiting The HomePage" do
+  scenario "Presented with header Thundrs" do
+    visit root_path
+    page.text.must_include "Thundrs!"
+  end
+
+  scenario "Visitor signs up for account" do
+    visit root_path
+    click_on "Sign up"
+
+    fill_in "Name", with: visitors(:visitor_1).name
+    fill_in "Email", with: visitors(:visitor_1).email
+    fill_in "Password", with: visitors(:visitor_1).password
+    fill_in "Password Confirmation", with: visitors(:visitor_1).password
```

Comment:
`visitors(:visitor_1).password` won't work, because the users table of the database won't have a `password` column. If we're using "password" for all the fixtures for consistency, this can just be `with: "password"`